packages/quick-edit/functions/_middleware.ts

```diff
@@ -16,6 +16,7 @@ export const onRequest = async ({
 						: "Solarflare Light",
 				"workbench.startupEditor": "none",
 				"editor.minimap.autohide": true,
+				"files.autoSave": "off",
```

Comment:
I was expecting the autosave to be disabled by default and it was enabled by us somewhere. However I couldn't find anywhere we set it to `afterDelay`. 🤔